### PR TITLE
Add all the mentioned libraries in survey

### PR DESCRIPTION
**Pull request by amitness**
Markdown listing of all the libraries and plugins mentioned in the survey result.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,196 @@
...
+- [json](https://docs.python.org/3/library/json.html): Read and write JSON files  
+- [jupyter](https://pypi.org/project/jupyter/): Jupyter notebooks  
+- [jupyterlab](https://pypi.org/project/jupyterlab/): An extensible environment for interactive and reproducible computing, based on the Jupyter Notebook and Architecture  
+- [keras](https://pypi.org/project/Keras/): High-level neural networks API, written in Python and capable of running on top of TensorFlow, CNTK, or Theano.  
+- [libffm](https://github.com/ycjuan/libffm): A Library for Field-aware Factorization Machines  
+- [libfm](http://www.libfm.org/): Factorization Machine Library  
+- [lightfm](https://github.com/lyst/lightfm): A Python implementation of LightFM, a hybrid recommendation algorithm.  
+- [lime](https://pypi.org/project/lime/): Local Interpretable Model-Agnostic Explanations for machine learning classifiers  
+- [matchzoo](https://matchzoo.readthedocs.io/en/master/): MatchZoo is a toolkit for text matching  
+- [matplotlib](https://pypi.org/project/matplotlib/): Matplotlib strives to produce publication quality 2D graphics
```
Comment (nissankarki) [inline README.md]: Remove this duplicate.

File: README.md [hunk 2]
```diff
@@ -1 +1,196 @@
...
+- [matchzoo](https://matchzoo.readthedocs.io/en/master/): MatchZoo is a toolkit for text matching  
+- [matplotlib](https://pypi.org/project/matplotlib/): Matplotlib strives to produce publication quality 2D graphics  
+- [memory-profiler](https://pypi.org/project/memory-profiler/): A module for monitoring memory usage of a python program  
+- [mongoengine](https://pypi.org/project/mongoengine/): MongoEngine is a Python Object-Document Mapper for working with MongoDB.  
+- [more_itertools](https://pypi.org/project/more-itertools/): More routines for operating on iterables, beyond itertools  
+- [multiprocessing-logging](https://pypi.org/project/multiprocessing-logging/): Logger for multiprocessing applications  
+- [newspaper](https://pypi.org/project/newspaper/): Simplified python article discovery & extraction.  
+- [nlopt](https://pypi.org/project/nlopt/): Library for nonlinear optimization, wrapping many algorithms for global and local, constrained or unconstrained, optimization  
+- [nltk](https://pypi.org/project/nltk/): Natural Language Toolkit  
+- [numpy](https://pypi.org/project/numpy/): NumPy is the fundamental package for array computing with Python.
```
Comment (nissankarki) [inline README.md]: Remove this duplicate

File: README.md [hunk 3]
```diff
@@ -1 +1,196 @@
...
+- [more_itertools](https://pypi.org/project/more-itertools/): More routines for operating on iterables, beyond itertools  
+- [multiprocessing-logging](https://pypi.org/project/multiprocessing-logging/): Logger for multiprocessing applications  
+- [newspaper](https://pypi.org/project/newspaper/): Simplified python article discovery & extraction.  
+- [nlopt](https://pypi.org/project/nlopt/): Library for nonlinear optimization, wrapping many algorithms for global and local, constrained or unconstrained, optimization  
+- [nltk](https://pypi.org/project/nltk/): Natural Language Toolkit  
+- [numpy](https://pypi.org/project/numpy/): NumPy is the fundamental package for array computing with Python.  
+- [opencv](https://pypi.org/project/opencv-python/): Wrapper package for OpenCV python bindings.  
+- [pandarallel](https://pypi.org/project/pandarallel/): An easy to use library to speed up computation (by parallelizing on multi CPUs) with pandas.  
+- [pandas](https://pypi.org/project/pandas/): Powerful data structures for data analysis, time series, and statistics
```
Comment (nissankarki) [inline README.md]: Remove this duplicate

File: README.md [hunk 4]
```diff
@@ -1 +1,196 @@
...
+- [gunicorn](https://pypi.org/project/gunicorn/): Production web server for Flask, Django apps  
+- [hungabunga](https://github.com/ypeleg/HungaBunga): HungaBunga: Brute-Force all sklearn models with all parameters using .fit .predict!  
+- [implicit](https://github.com/benfred/implicit): Fast Python Collaborative Filtering for Implicit Feedback Datasets  
+- [interpret](https://pypi.org/project/interpret/): Fit interpretable models. Explain blackbox machine learning.  
+- [ipython](https://pypi.org/project/ipython/): IPython: Productive Interactive Computing  
+- [itertools](https://docs.python.org/2/library/itertools.html): Functions creating iterators for efficient looping  
+- [json](https://docs.python.org/3/library/json.html): Read and write JSON files  
+- [jupyter](https://pypi.org/project/jupyter/): Jupyter notebooks  
+- [jupyterlab](https://pypi.org/project/jupyterlab/): An extensible environment for interactive and reproducible computing, based on the Jupyter Notebook and Architecture  
+- [keras](https://pypi.org/project/Keras/): High-level neural networks API, written in Python and capable of running on top of TensorFlow, CNTK, or Theano.
```
Comment (nissankarki) [inline README.md]: Remove this duplicate

**Comment by amitness** [comment]
Changes have been done @nissankarki. Please review.

**Pull request merged**